### PR TITLE
Header Bar Config Tutorial

### PR DESCRIPTION
**Pull request by oovm**
Need Fix:
- [x] the meaning of `exact` in `nav` setting
- [x] chinese punctuation format
- [x] add commit id for new page

### CODE REVIEW

File: docs/_posts/2019-03-23-header-config-cn.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+tags:
+  - cn
+date: 2019-03-23
+title: 主题标题栏的设置
+header-image:
+  - /assets/img/header-image-01.jpg
+  - /assets/img/header-image-02.jpg
+  - /assets/img/header-image-03.jpg
+header-title: true
+---
```
Comment (meteorlxy) [inline docs/_posts/2019-03-23-header-config-cn.md]: `vssue-id: 2` is required for zh docs

File: docs/_posts/2019-03-23-header-config-cn.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+category: demo
+tags:
+  - cn
+date: 2019-03-23
```
Comment (meteorlxy) [inline docs/_posts/2019-03-23-header-config-cn.md]: The `date`s in the docs are not for the exact created time, but for the display order.

(we could also support `top: true` frontmatter to make a certain post placed on the top)

**Review by meteorlxy — CHANGES_REQUESTED** [review]
Greak thanks for you contribution ❤️ 

It still need some enhancement before merging

**Comment by oovm** [comment]
what's the meaning of `exact`?
https://github.com/meteorlxy/vuepress-theme-meteorlxy/pull/30/commits/d3e5abdbed6f70c45d3970a9f92d7bad28accd0a#diff-993c8acbd6da41f3c37bece6640c06bcR52

**Comment by meteorlxy** [comment]
@GalAster 

It's the `exact` prop of `RouterLink`:

https://router.vuejs.org/api/#exact

File: docs/_posts/2019-03-23-header-config-cn.md (hunk 2 above)
Comment (oovm) [reply docs/_posts/2019-03-23-header-config-cn.md]: ok, i'll use `2019-01-20`, the day before the en, to keep the same part together.

**Comment by oovm** [comment]
@meteorlxy please review

File: docs/_posts/2019-01-21-header-config-en.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+![](/assets/img/header-introduce.png)
+
+The header bar of the blog can be divided into following sections:
+
+- [theme-title](/posts/2019/03/23/header-config-cn.html#theme-title)
+- [theme-nav](/posts/2019/03/23/header-config-cn.html#theme-navigation)
+- [header-title](/posts/2019/03/23/header-config-cn.html#header-title)
+- [header-image](/posts/2019/03/23/header-config-cn.html#header-image)
+
```
Comment (meteorlxy) [inline docs/_posts/2019-01-21-header-config-en.md]: We can use `<TOC />` for the table of content

File: docs/_posts/2019-03-23-header-config-cn.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
+---
+category: demo
+tags:
+  - cn
```
Comment (meteorlxy) [inline docs/_posts/2019-03-23-header-config-cn.md]: `cn` => `zh`

File: docs/_posts/2019-03-23-header-config-cn.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+date: 2019-01-20
+title: 主题标题栏的设置
```
Comment (meteorlxy) [inline docs/_posts/2019-03-23-header-config-cn.md]: 不如和英文版一样短点就叫 `标题栏的设置` 吧 （其实不改也OK 😅

File: docs/_posts/2019-03-23-header-config-cn.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+<!-- more -->
+
+![](/assets/img/header-introduce.png)
+
+博客的标题栏分为这样几个部分：
+
+- [theme-title](/posts/2019/03/23/header-config-cn.html#theme-title)    | 整个博客的名称         
+- [theme-nav](/posts/2019/03/23/header-config-cn.html#theme-navigation) | 导航栏自定义页面       
+- [header-title](/posts/2019/03/23/header-config-cn.html#header-title)  | 文章或页面的小标题     
+- [header-image](/posts/2019/03/23/header-config-cn.html#header-image)  | 文章或页面标题的背景图
```
Comment (meteorlxy) [inline docs/_posts/2019-03-23-header-config-cn.md]: 同上

File: docs/_posts/2019-01-21-header-config-en.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+      { text: 'Posts', link: '/posts/', exact: false },
+      { text: 'Custom Pages', link: '/custom-pages/', exact: false },
+      { text: 'Github', link: 'https://github.com/meteorlxy/vuepress-theme-meteorlxy' },
+    ],
+  }
+}
+```
+
+- `link` can be an internal relative link (starting with `/`) or a full external link.
+- `exact` refer to [Vue-Router](https://router.vuejs.org/api/#exact)
```
Comment (meteorlxy) [inline docs/_posts/2019-01-21-header-config-en.md]: refers

File: docs/_posts/2019-03-23-header-config-cn.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+---
+```
+
+::: tip
+如果是本地图片，默认需要上传至 `public` 文件夹下才会在构建的时候打包进去。
+
+完整的路径名类似于 `.vuepress/public/assets/img/header-image-01.jpg`。
+:::
+
+如果感觉图片的宽度不合适，那么可以在 `.vuepress/styles/palette.styl` 中调整：
```
Comment (meteorlxy) [inline docs/_posts/2019-03-23-header-config-cn.md]: 应该是 高度不合适 吧

File: docs/_posts/2019-03-23-header-config-cn.md (hunk 6 above)
Comment (meteorlxy) [reply docs/_posts/2019-03-23-header-config-cn.md]: 另外用了 `<TOC />` 之后不用担心标题 anchor 中英文问题，下面的小标题也可以直接换成中文的

**Review by meteorlxy — CHANGES_REQUESTED** [review]
感谢老铁，另外刚才发了个新版本把 TOP 功能加上了，可以顺手合进来先

File: docs/_posts/2019-01-21-header-config-en.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+---
+```
+
+::: tip
+If it is a local image, it should be uploaded to the `public` folder by default, and then it will be packaged in the build.
+
+The full path is similar to `.vuepress/public/assets/img/header-image-01.jpg`
+:::
+
+If you feel that the heights of the image is not appropriate, you can adjust it in `.vuepress/styles/palette.styl`:
```
Comment (meteorlxy) [inline docs/_posts/2019-01-21-header-config-en.md]: height

**Review by meteorlxy — CHANGES_REQUESTED** [review]
typo

**Comment by oovm** [comment]
- 2019-01-21: header-config-en
- 2019-01-20: header-config-zh

`disable-comments.md` moved from 2019-01-20 to 2019-01-25 for avoiding conflict.

assign 2019-01-24 for `comment-config-cn` in feature PR

**Review by meteorlxy — APPROVED** [review]

**Comment by meteorlxy** [comment]
:heart:

**Pull request merged**